src/unix/core.c

```diff
@@ -63,6 +63,9 @@ extern char** environ;
 #endif /* !(defined(__APPLE__) && !TARGET_OS_IPHONE) */
 
 
+#if defined(__linux__)
+# define uv__accept4 accept4
+#endif
```

Comment:
Can you fold this into the `#if defined(__linux__)` block around line 90 to keep things from getting too scattered around?

Reply:
> Can you fold this into the `#if defined(__linux__)` block around line 90 to keep things from getting too scattered around?

done

Reply:
I added a space before `#include` to keep it consistent with other codes